### ISSUE TITLE
[Feature Request]: Backup Power SOC entity

### ISSUE THREAD

**Issue by vwtuner** [wlcrs/huawei_solar#401]
### Describe your feature request

„Backup Batterie Kapazität“( Reservierte Batterie Kapazität)
Einstellung bitte hinzufügen 
![IMG_1560](https://github.com/wlcrs/huawei_solar/assets/121826531/c1e4ab0a-1b0e-40c8-9dd7-786c60375293)

### Proper usage

- [X] I confirm that this is not a bug report
- [X] I confirm that this feature request is within the stated scope of the integration

**Comment by streamthing** [comment]
Write in English!!!

**Comment by streamthing** [comment]
@vwtuner Write in English!!!

**Comment by vwtuner** [comment]
"Backup Battery Capacity" (Reserved Battery Capacity)
Please add setting

**Comment by WiebKastanje** [comment]
Hi @vwtuner, I'm too interested in this described Reserved Battery Capacityf eature. We have to make clear, in plain english, what is needed.
Please provide the link to Huawei document are you referring to, describing the "netzentkoppelte modes"? And abbildung 3-15?   
Thank you very much.

**Comment by vwtuner** [comment]
[Back up Anleitung .pdf](https://github.com/wlcrs/huawei_solar/files/11929948/Back.up.Anleitung.pdf)

![IMG_1740](https://github.com/wlcrs/huawei_solar/assets/121826531/64d25f92-bd7e-452e-a382-3259ae433ee6)


Here is the English version.

**Comment by WiebKastanje** [comment]
The feature request is: 

Kindly add input field to set the ""Backup power SOC" parameter to a certain percentage. Maximum = 100%. 
Default value = NAN
![image](https://github.com/wlcrs/huawei_solar/assets/46934766/3df264ee-9116-4543-a9c3-6ac8e0d1ae52)

This field is an off-grid control parameter, specially important for users with a Huawei backup-box connected to the inverter, which provides energy when utility powergrid is down.

Thank you.

**Comment by vwtuner** [comment]
Hi, thank you very much for your effort to add the backup soc.
unfortunately there is always an error when entering a value.
when I want to activate loading from the network, an error also occurs (which worked before)Gesendet mit der mobilen Mail AppAm 10.07.23 um 15:43 schrieb Thijs W.
                
            
            
                
                    Von: "Thijs W." ***@***.***>Datum: 10. Juli 2023An: "wlcrs/huawei_solar" ***@***.***>Cc: "vwtuner" ***@***.***>,"Mention" ***@***.***>Betreff: Re: [wlcrs/huawei_solar] [Feature Request]: Backup Power SOC entity (Issue #401)
                
                
This feature was implemented in 1.2.6 beta 8, but as I cannot detect if a battery box is installed the entity is disabled by default.



Please also note that the valid range of this register is influenced by the Capacity Control SOC (when enabled). I haven't implemented any checks on whether a valid value is being configured and as I do not own a backup box I have no idea what happens when you do so.
Use at your own risk.

**Comment by vwtuner** [comment]
![IMG_1995](https://github.com/wlcrs/huawei_solar/assets/121826531/e85aa9cb-117a-40a9-bd4e-0d9c7c4d4be5)
The SOC backup is from 0-100% 
Not from 20-100%

**Comment by wlcrs** [comment]
Please test again with 1.2.6 beta 9

**Comment by vwtuner** [comment]
currently it works as it should, thank you for the great work. may I ask where the error was?
